python/tank/deploy/tank_commands/install.py

```diff
@@ -140,22 +162,33 @@ def run_interactive(self, log, args):
             log.info("> tank install_app Asset tk-shell tk-multi-loader")
             log.info("")
             log.info("")
-            
+            log.info("Comment and structure preserving mode")
+            log.info("----------------------------------------")
```

Comment:
Nitpick: I don't know how we do else where, but since the console uses a fixed font maybe the line of dashes should be  as wide as the text above it.